apps/juror-digital/jd-public/prod.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   values:
     nodejs:
-      ingressHost: 'reply-jury-summons.service.gov.uk'
+      # ingressHost: 'reply-jury-summons.service.gov.uk'
```

Comment:
Are we leaving Juror-digital up after the new service go live? if we are then may want to replace this with something else? Like a juror-digital.platform.hmcts.net?

Ignore this if JD will be decommed post go live.

Reply:
I am OK to leave it simply without any ingress - it will be decommend post ELS